mne/io/ctf/info.py

```diff
@@ -68,7 +68,11 @@ def _convert_time(date_str, time_str):
         else:
             break
     else:
-        raise RuntimeError("Illegal date: %s" % date)
+        raise RuntimeError("""Illegal date: %s.  
+If the language of the date does not correspond to your local machine's 
+language try to set the locale to the language of the date string: 
+locale.setlocale(locale.LC_ALL, 'en_US')""" % date_str)
```

Comment:
we don't usually use `"""`, so can you redo this with single quotes? If you want to save lines you can change it to:
```
        raise RuntimeError(
            'Illegal date: %s.\nIf the language of the date does not '
            'correspond to your local machine\'s language try to set the '
            'locale to the language of the date string:\n'
            'locale.setlocale(locale.LC_ALL, "en_US")' % date_str)
```